plejd/PlejdApi.js

```diff
@@ -369,30 +370,46 @@ class PlejdApi {
 
           this.deviceRegistry.addOutputDevice(outputDevice);
         }
-      }
-
-      // What should we do with inputs?!
-      // if (outputDevice.typeName === 'WPH-01') {
-      //   // WPH-01 is special, it has two buttons which needs to be
-      //   // registered separately.
-      //   const inputs = this.siteDetails.inputAddress[deviceId];
-      //   const first = inputs[0];
-      //   const second = inputs[1];
-
-      //   this.deviceRegistry.addPlejdDevice({
-      //     ...outputDevice,
-      //     id: first,
-      //     name: `${device.title} left`,
-      //   });
-
-      //   this.deviceRegistry.addPlejdDevice({
-      //     ...outputDevice,
-      //     id: second,
-      //     name: `${device.title} right`,
-      //   });
-      // } else {
-      //   this.deviceRegistry.addPlejdDevice(outputDevice);
-      // }
+      } else {
+        // The device does not have an output. It can be assumed to be a WPH-01 or a WRT-01
+        // Filter inputSettings for available buttons
+        const inputSettings = this.siteDetails.inputSettings.filter(
```

Comment:
Related to above - maybe this is enough of a condition? All inputs that meet this criteria will propagate over BLE? 
(looks suspiciously WPH-01-specific though. Guess from me is that WRT-01 will have a different buttonType...

Reply:
I was thinking so. As soon as I get logs from a WRT-01 I may have to update this.
But there are more places to fix to support WRT-01 I think.